src/main/java/store/config/InitialDataLoader.java

```diff
@@ -0,0 +1,106 @@
+package store.config;
+
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayDeque;
+import java.util.List;
+import java.util.Queue;
+import store.error.ErrorMessage;
+import store.error.exception.InvalidFailInitialDataLoadException;
+import store.product.domain.Product;
+import store.product.domain.ProductInfo;
+import store.product.domain.ProductRepository;
+import store.promotion.domain.Promotion;
+import store.promotion.domain.PromotionDateTime;
+import store.promotion.domain.PromotionRepository;
+import store.promotion.domain.PromotionType;
+import store.util.StringParser;
+
+public class InitialDataLoader {
+
+    private static final String PATH_PROMOTIONS = "promotions.md";
+    private static final String PATH_PRODUCTS = "products.md";
+    private static final String DELIMITER_COMMA = ",";
```

Comment:
p4

```suggestion
    private static final String DELIMITER = ",";
```
구분자는 변경가능성이 크다고 조심스럽게 생각해 봅니다.
 만약 구분자가 변경된다면  변수 이름도 변경해야하기 때문에 변경에도 유연하게 대처하기 위해 DELMITER로 변수명을 명명하는 것도 하나의 방법이 될것 같습니다. 😄

Reply:
좋은 의견 감사해요 ㅎㅎ!

저는 추후에 다른 구분자가 추가될 수도 있으니까 저렇게 네이밍을 했어요 !

Reply:
우왁 채택이되다닝!